### ISSUE TITLE
Admin Content Search enhancement

### ISSUE THREAD

**Issue by GioviQ** [OrchardCMS/Orchard#6022]
I would like search capabilities in Admin Search Content like front-end search.
E.g. at the moment I have to find all content not containing some words in the body part.

**Comment by mattcowen** [comment]
Have you tried the Admin Content Search feature that you can enable?

**Comment by GioviQ** [comment]
But I am just talking about enhancing that feature. My request is not clear?

**Comment by mattcowen** [comment]
Sorry, I misread your request. You want to be able to negate your search query which you can't do currently in the Admin Content Search. I have tried to do this in the front-end search with "-" preceding my search text but this doesn't work for me. You seem to suggest that it does work for you in the front-end so, if that is the case, how are you doing it?

It is something that Lucene supports along with wildcards and other query syntax but I can't seem to get any of them working in Orchard.

**Comment by mattcowen** [comment]
It appears the "-" is being stripped off the query by the QueryParser on line 84 of LuceneSearchBuilder. Could be related to a known issue with special characters....

https://github.com/OrchardCMS/Orchard/issues/3887

**Comment by GioviQ** [comment]
in the admin back end (not Admin Content Search) you can design searches for projections (Search link in the left side of back end). Such part has just the features with the advanced filters I need.

**Comment by sebastienros** [comment]
@mattcowen it's stripped out on purpose, this is the goal of QueryParser. I just decided that whatever was typed in the box was the list of terms to search, not a technical query per se. I don't think we should support custom verbs in it. But we can always have an advanced search with more fields like excluded terms, sort criteria, content types, ...

**Comment by mattcowen** [comment]
Yes perhaps too technical for the average user. An advanced search sounds good way to go.

For those interested, an interesting discussion on this topic in the core Lucene project... https://issues.apache.org/jira/browse/LUCENE-4378

**Comment by GioviQ** [comment]
In fact it would be enough to me to leave Admin Content Search the same and add an "edit content" link for each item result in project search preview.

**Comment by sebastienros** [comment]
@GioviQ There is already such things, no? This is just the same standard content item list view with the actions on each item, including the Edit. Or just click the content item's title link.

**Comment by GioviQ** [comment]
@sebastienros I do not find any link to edit the items in this preview (see screenshot). 
![muscoli info gestione ricerche](https://cloud.githubusercontent.com/assets/7103057/11130165/eaa11312-8983-11e5-939a-24fba739f74a.png)

This preview contains all contet item without word "respirazione" in the body part.
